Lifetime extension fix (#33)
arfc/bae-synergistic: sensitivity.tex

Increase in the breeding ratio of \gls{ASTRID} reactors
decreases the total reprocessing demand, since less

> \gls{UNF} is reprocessed to extract the same amount of

Reviewer: ```suggestion
\gls{UNF} must be reprocessed to extract the same amount of
```

> plutonium. Also, less \gls{LWR} \gls{UNF} is reprocessed

Reviewer: ```suggestion
plutonium. Additionally,
```

> because \glspl{ASTRID} become independent quicker due to

Reviewer: ```suggestion 
\glspl{ASTRID} become independent more quickly due to
```

higher breeding of plutonium.

> Figure \ref{fig:br_fuel} shows the fuel loading in \glspl{ASTRID}

Reviewer: ```suggestion
Figure \ref{fig:br_fuel} shows the relationship between breeding ratio and fuel loading in \glspl{ASTRID}.
```

> and how it changes with the increase of breeding ratio of \glspl{ASTRID}.

Reviewer: ```suggestion
```

    \label{fig:pow_diff}
\end{figure}

Figure \ref{fig:ext_fuel} shows the change in \gls{ASTRID} fuel loading with
\gls{LWR} lifetime extension. The \gls{ASTRID} fuel loaded with plutonium
from \gls{LWR} \gls{UNF} convey the corresponding \gls{LWR} \gls{UNF} reprocessing demand.
The change in \gls{ASTRID} deployment alters \gls{ASTRID} fuel loading patterns,

> but does not increase the \gls{LWR} \gls{UNF} demand significantly.

Author: changed to:
```The change in \gls{ASTRID} deployment alters \gls{ASTRID} fuel loading patterns.
 However, increasing the \gls{LWR} lifetimes
does not increase the \gls{LWR} \gls{UNF} demand significantly (less than 4\%
for the 20-year extension case),
because \glspl{ASTRID} become self-sustaining at similar times after the first
\gls{ASTRID} deployment.```

	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/10yr.png}}\quad
	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/20yr.png}}
	\caption{\gls{ASTRID} fuel loading patterns are altered by changes in \gls{ASTRID} deployment
			 caused by the lifetime extension of \glspl{LWR}.}
	\label{fig:ext_fuel}
\end{figure}

The quantitative effects of
\gls{LWR} lifetime extensions are shown in table \ref{tab:ext_met}.
The differences were calculated using the same equation used for the
breeding ratio study. 
Since \gls{LWR} lifetime extensions shorten the span of \gls{ASTRID} operations, less \gls{ASTRID} fuel is needed when \gls{LWR} lifetimes are extended.
Therefore, it is not fair to compare the mass of total \gls{UNF} reprocessed,
since less plutonium is extracted from \gls{UNF}. Instead, we compared the
average reprocessing values and the total amount of \gls{LWR} \gls{UNF} reprocessed.
There is not a significant difference (less than $1\%$ for the 20-year extension case)
in the amount of \gls{LWR}
\gls{UNF} reprocessed. The delay in \gls{ASTRID} deployment
spreads out the \gls{LWR} \gls{UNF} reprocessing demand,

> thereby dramatically reduces ($39\%$ for the 20-year extension case)

Reviewer: ```suggestion
thereby dramatically reducing ($39\%$ for the 20-year extension case)
```

    \label{fig:pow_diff}
\end{figure}

Figure \ref{fig:ext_fuel} shows the change in \gls{ASTRID} fuel loading with
\gls{LWR} lifetime extension. The \gls{ASTRID} fuel loaded with plutonium
from \gls{LWR} \gls{UNF} convey the corresponding \gls{LWR} \gls{UNF} reprocessing demand.
The change in \gls{ASTRID} deployment alters \gls{ASTRID} fuel loading patterns,
but does not increase the \gls{LWR} \gls{UNF} demand significantly.

\begin{figure}[!ht]
	\centering
	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/base.png}}\quad
	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/5yr.png}}\\
	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/10yr.png}}\quad
	\subfloat{\includegraphics[width=.45\textwidth]{./images/sensitivity/20yr.png}}

> 	\caption{\gls{ASTRID} fuel loading patters are altered by changes in \gls{ASTRID} deployment

Reviewer: ```suggestion
	\caption{\gls{ASTRID} fuel loading patterns are altered by changes in \gls{ASTRID} deployment
```

			 caused by the lifetime extension of \glspl{LWR}.}
	\label{fig:ext_fuel}
\end{figure}

The quantitative effects of
\gls{LWR} lifetime extensions are shown in table \ref{tab:ext_met}.
The differences were calculated using the same equation used for the

> breeding ratio study. Different metrics were studied for the lifetime

Reviewer: ```suggestion
breeding ratio study. 
```

> extension study since there are less \gls{ASTRID} fuel produced due to

Reviewer: ```suggestion
Since \gls{LWR} lifetime extensions shorten the span of \gls{ASTRID} operations, less \gls{ASTRID} fuel is needed when \gls{LWR} lifetimes are extended.
```

> the shortening of \gls{ASTRID} operations caused by \gls{LWR} lifetime

Reviewer: ```suggestion
```

> extension. Therefore, it is not fair to compare the mass of total \gls{UNF} reprocessed,

Reviewer: ```suggestion
Therefore, it is not fair to compare the mass of total \gls{UNF} reprocessed,
```

since less plutonium is extracted from \gls{UNF}. Instead, we compared the

> average reprocessing values and the total amount of \gls{LWR} \gls{UNF} reprocessed.

Reviewer: I find the last many sentences in this paragraph very confusing.

Author: I tried to clarify it, but let me know if it still is:
```
There is not a significant increase (less than $4\%$ for the 20-year extension case)
in the amount of \gls{LWR}
\gls{UNF} reprocessed. The delay in \gls{ASTRID} deployment
spreads out the \gls{LWR} \gls{UNF} reprocessing demand,
thereby dramatically reduces ($21\%$ for the 20-year extension case)
the average monthly \gls{LWR} \gls{UNF} reprocessing demand.
```
